proto/monitor_service.proto

```diff
@@ -54,25 +54,29 @@ message AnalyzeHeapResponse {
 // Back pressure
 message GetBackPressureRequest {}
 
-message BackPressureInfo {
-  uint32 actor_count = 3;
-  // Sum of output blocking duration of all actors
-  double value = 4;
+message ChannelStats {
+  uint32 actor_count = 1 [json_name = "n"];
+  // Sum of output blocking duration of all actors in nanoseconds
+  double blocking_duration = 2 [json_name = "b"];
+  // Sum of input row count of all actors
+  uint64 input_row_count = 3 [json_name = "i"];
+  // Sum of output row count of all actors
+  uint64 output_row_count = 4 [json_name = "o"];
```

Comment:
Could you explain why we add `json_name` here?

Reply:
Because this message is repeated many times in the JSON response, use shorter key to reduce message size. Let me comment it in code as well.

Reply:
So I suppose the generated code for JS can also correctly handle this. Sounds good.